metricbeat/module/consul/_meta/fields.yml

```diff
@@ -0,0 +1,10 @@
+- key: consul
+  title: "consul"
+  description: >
+    Consul module
+  release: beta
+  fields:
+    - name: consul
+      type: group
+      description: >
```

Comment:
Empty description, add one or remove the field.

Reply:
Field removed